packages/salesforcedx-vscode-lwc/package.nls.json

```diff
@@ -14,5 +14,9 @@
   "force_lightning_lwc_test_run_current_file_text": "SFDX: Run Current Lightning Web Component Test File",
   "force_lightning_lwc_test_debug_current_file_text": "SFDX: Debug Current Lightning Web Component Test File",
   "force_lightning_lwc_test_start_watching_text": "SFDX: Start Watching Lightning Web Component Test",
-  "force_lightning_lwc_test_stop_watching_text": "SFDX: Stop Watching Lightning Web Component Test"
+  "force_lightning_lwc_test_stop_watching_text": "SFDX: Stop Watching Lightning Web Component Test",
+  "force_lightning_lwc_preferences": "Lightning Web Components Preview",
+  "force_lightning_lwc_remember_device_description": "Remember most recently used mobile device target.",
+  "force_lightning_lwc_mobile_log_level": "Log level used when calling SFDX Preview on Mobile command.",
+  "force_lightning_lwc_enable_mobile": "Warning: This feature is not publicly available."
```

Comment:
This should probably preface with what it does as well: "Preview Lightning Web Component on Mobile.  Warning:..."

Reply:
Just spoke to Richard about this and he is putting something together.  On a related note, this might be in the wrong place.   I just found there is a `Salesforce Feature Previews` settings configuration under the core library.  I will check with Luis, but I think the flag belongs there.